### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by simmel**
What do you think?

### CODE REVIEW

**Comment by simmel** [comment]
I'm also going to create a PR so one can just `pip install collectd-prometheus` but that might come next week.

File: README.md [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    ModulePath "/usr/lib/collectd/python" # This is the folder we downloaded collectd-prometheus.py into before
+    Import "collectd-prometheus"
+    <Module "collectd-prometheus">
+       Interval 30 # How often to scrape metrics. This is the default, can be omitted
+       <Process>
+           Process "mycoolservice" # Name this instance, e.g. after what service you're scraping
+           Protocol "http" # This is default, can be omitted
+           Host "127.0.0.1" # This is default, can be omitted
+           Port "8080" # This is default, can be omitted
+           Filter "only|these|metrics" # A regex which matches the names of the metrics you only want to include
```
Comment (ryarnyah) [inline README.md]: You can have multiple `Filter` if needed

**Review by ryarnyah — COMMENTED** [review]
Just awesome! thank you!

File: README.md (hunk 1 above)
Comment (simmel) [reply README.md]: Ooh, that's nice! Now that you say it I saw a loop around it! I'll fix.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    Import "collectd-prometheus"
+    <Module "collectd-prometheus">
+       Interval 30 # How often to scrape metrics. This is the default, can be omitted
+       <Process>
+           Process "mycoolservice" # Name this instance, e.g. after what service you're scraping
+           Protocol "http" # This is default, can be omitted
+           Host "127.0.0.1" # This is default, can be omitted
+           Port "8080" # This is default, can be omitted
+           Filter "these|two" # A regex which matches the names of the metrics you only want to include
+           Filter "metrics" # You can even specify multiple regexes
```
Comment (simmel) [inline README.md:41]: How is that?

Comment (simmel) [reply README.md:41]: Actually, I change it back to the original "regex" since it made more sense.

**Review by ryarnyah — APPROVED** [review]

**Pull request merged**